README.md

```diff
@@ -533,6 +571,63 @@ project.getMetadata(function(err, metadata) {
 ```
 
 
+## Google Cloud Speech (Limited Preview)
+
+> **This is a Limited Preview release of Google Cloud Speech.** This feature is not covered by any SLA or deprecation policy and may be subject to backward-incompatible changes.
+
+- [API Documentation][gcloud-speech-docs]
+- [Official Documentation][cloud-speech-docs]
+
+```js
+var gcloud = require('gcloud');
+
+// Authorizing on a per-API-basis. You don't need to do this if you auth on a
+// global basis (see Authorization section above).
+
+var speech = gcloud.speech({
+  projectId: 'my-project',
+  keyFilename: '/path/to/keyfile.json'
+});
+
+// Detect the speech in an audio file.
+speech.detect('./audio.raw', {
+  encoding: 'LINEAR16',
+  sampleRate: 16000
+}, function(err, result) {
+  // result = {
+  //   transcript: 'how old is the Brooklyn Bridge',
+  //   confidence: 0.97234234234
+  // ]
+});
+```
+
+```js
+// Detect the speech in an audio file stream.
+var initialRequest = {
+  encoding: 'LINEAR16',
+  sampleRate: 16000,
+  continuous: true
+};
+
+var stream = speech.recognize(initialRequest);
+  .on('data', function (recognizeResponse) {
+    console.log(recognizeResponse);
+  });
+
+stream.write({
+  audioRequest: {
```

Comment:
The config data is probably what @jgeewax was talking about here:

```js
speech.recognize({ info about the data, including a URI for where to get it or the raw bytes })
// and
speech.recognizeFromStream({ info about the data, no URI / bytes })
```

I think where we want to be is something like:

#### `SyncRecognize`
```js
var config = {
  info about this data: {},
  uri or bytes (data): {},
  sync: true
}

speech.recognize(config, function(err, syncResponse) {})
```

#### `AsyncRecognize`
```js
var config = {
  info about this data: {},
  uri or bytes (data): {}
  // sync: true ( note that this is commented out, so not part of the config )
}

speech.recognize(config, function(err, job) {
  job
    .on('error', function(err) {})
    .on('complete', function(asyncRecognizeResponse) {})
})
```

#### `StreamingRecognize`
```js
var config = { info about this data }

fs.createReadStream('./my-file')
  .pipe(speech.recognize(config))
  .on('error', function(err) {}
  .on('response', function(resp) {})
```

Does this sound right / plausible?